.buildkite/docker/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM rust:1.65
+FROM rust:1.68
 
 RUN rustup component add rustfmt && \
 	rustup component add clippy
 
-RUN cargo install cargo-tarpaulin
+# RUN cargo install cargo-tarpaulin
```

Comment:
Did you leave this here intentionally?

Reply:
Ya, I want to bring it back if it works again - or swap it for a different one maybe.